### PR TITLE
Fix #3461 : Merged App Version fragment and app version activity into individual files

### LINKED ISSUE #3461: Merge app_version_activity and app_version_fragment into individual xml file
Currently there are 2 versions of `app_version_activity.xml` file merge it into single xml file.
Currently there are 3 versions of `app_version_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by Rohit1173**
## Explanation
Fix #3461 
There were 3 files of app_version_fragment, I have and I compared them in the given website and added the specific changes in their respective dimens.xml files. 
There were 2 files of app_version_activity, and both were identical so I have deleted the Landscape file
Here are the screenshots of the app before and after the above-mentioned changes
|BEFORE|AFTER|
|---------|-------|
|before-1|after-1|
|before-2|after-2|
|before-3|after-3|
|before-4|after-4|

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Rohit1173** [comment]
@anandwana001 @rt4914 PTAL

File: .idea/deploymentTargetDropDown.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+          <Key>
+            <type value="VIRTUAL_DEVICE_PATH" />
+            <value value="C:\Users\sairo\.android\avd\Pixel_C_API_30.avd" />
+          </Key>
+        </deviceKey>
+      </Target>
+    </targetSelectedWithDropDown>
+    <timeTargetWasSelectedWithDropDown value="2021-08-14T16:18:34.004930400Z" />
+  </component>
+</project>
```
Comment (prayutsu) [inline .idea/deploymentTargetDropDown.xml]: Revert these changes

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,9 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/sairo/opensource/oppia-android/app/src/main/res/layout/app_version_fragment.xml" value="0.11246200607902736" />
+      </map>
+    </option>
+  </component>
```
Comment (prayutsu) [inline .idea/misc.xml]: Revert the changes in this file.

File: .idea/runConfigurations.xml [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-  <component name="RunConfigurationProducerService">
-    <option name="ignoredProducers">
-      <set>
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
-      </set>
-    </option>
-  </component>
-</project>
```
Comment (prayutsu) [inline .idea/runConfigurations.xml]: We don't want to delete this file, please restore it

File: app/src/main/res/values-land/dimens.xml [hunk 4]
```diff
@@ -270,4 +270,10 @@
   <dimen name="coming_soon_topic_promoted_list_view_padding_start">0dp</dimen>
   <dimen name="coming_soon_topic_Promoted_list_view_padding_end">0dp</dimen>
 
+  <!-- APP VERSION FRAGMENT -->
```
Comment (prayutsu) [inline app/src/main/res/values-land/dimens.xml]:
```suggestion
  <!-- App Version -->
```

Ditto elsewhere.

**Review by prayutsu — CHANGES_REQUESTED** [review]
Left some comments, PTAL @Rohit1173

File: .idea/runConfigurations.xml (hunk 3 above)
Comment (Rohit1173) [reply .idea/runConfigurations.xml]: Done

File: .idea/misc.xml (hunk 2 above)
Comment (Rohit1173) [reply .idea/misc.xml]: Done

File: .idea/deploymentTargetDropDown.xml (hunk 1 above)
Comment (Rohit1173) [reply .idea/deploymentTargetDropDown.xml]: Done

File: app/src/main/res/values-land/dimens.xml (hunk 4 above)
Comment (Rohit1173) [reply app/src/main/res/values-land/dimens.xml]: Done

**Comment by Rohit1173** [comment]
@prayutsu PTAL

**Comment by prayutsu** [comment]
@Rohit1173 You should never mark conversation resolved if you didn't initiate the comment. They should only be resolved by the reviewers.
I am marking them unresolved and will resolve them again if I find that the change has been done.

**Review by prayutsu — APPROVED** [review]
LGTM, Thanks @Rohit1173

**Comment by Rohit1173** [comment]
> LGTM, Thanks @Rohit1173



> @Rohit1173 You should never mark conversation resolved if you didn't initiate the comment. They should only be resolved by the reviewers.
> I am marking them unresolved and will resolve them again if I find that the change has been done.

I will make sure this doesn't happen again
Thanks,

File: app/src/main/res/layout/app_version_fragment.xml [hunk 5]
```diff
@@ -24,9 +24,9 @@
         android:layout_height="wrap_content"
         android:background="@drawable/selector_app_version"
         android:fontFamily="sans-serif"
-        android:paddingStart="16dp"
+        android:paddingStart="@dimen/app_version_padding_start"
```
Comment (rt4914) [inline app/src/main/res/layout/app_version_fragment.xml]: Rename this to `app_version_text_view_padding_start `

File: app/src/main/res/layout/app_version_fragment.xml [hunk 6]
```diff
@@ -24,9 +24,9 @@
...
         android:paddingTop="20dp"
-        android:paddingEnd="16dp"
+        android:paddingEnd="@dimen/app_version_textview_margin_end"
```
Comment (rt4914) [inline app/src/main/res/layout/app_version_fragment.xml]: Rename this to `app_version_text_view_padding_end`

If you are setting padding attribute then the variable name should contain padding in it instead of margin.

File: app/src/main/res/layout/app_version_fragment.xml [hunk 7]
```diff
@@ -39,7 +39,7 @@
         android:id="@+id/app_info_image_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
+        android:layout_marginStart="@dimen/app_version_imageview_margin_start"
```
Comment (rt4914) [inline app/src/main/res/layout/app_version_fragment.xml]: Nit: `app_version_image_view_margin_start`

File: app/src/main/res/layout/app_version_fragment.xml (hunk 6 above)
Comment (rt4914) [reply app/src/main/res/layout/app_version_fragment.xml]: Also this has been used incorrectly as the value of `app_version_textview_margin_end ` is `28dp` not `16dp`.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Rohit1173 Please take the values again it looks like the values have been used incorrectly.

File: app/src/main/res/layout/app_version_fragment.xml (hunk 5 above)
Comment (Rohit1173) [reply app/src/main/res/layout/app_version_fragment.xml]: Done

File: app/src/main/res/layout/app_version_fragment.xml (hunk 6 above)
Comment (Rohit1173) [reply app/src/main/res/layout/app_version_fragment.xml]: I did a small mistake by writing the app_version_textview_margin_end twice.
I had corrected it now.

File: app/src/main/res/layout/app_version_fragment.xml (hunk 7 above)
Comment (Rohit1173) [reply app/src/main/res/layout/app_version_fragment.xml]: Done

**Comment by Rohit1173** [comment]
@rt4914 PTAL

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 8]
```diff
@@ -295,4 +295,10 @@
   <dimen name="topic_practice_footer_view_margin_top">24dp</dimen>
   <dimen name="topic_practice_footer_view_margin_bottom">0dp</dimen>
 
+  <!-- App Version -->
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/dimens.xml:298]: You have deleted file from `res/layout-sw600dp` which is used for tablet (portait+landscape). But you have introduced these values only in `landscape-dimens` file. Please add same values in `values-sw600dp-port` too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Rohit1173 PTAL

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 8 above)
Comment (Rohit1173) [reply app/src/main/res/values-sw600dp-land/dimens.xml:298]: Done

**Comment by Rohit1173** [comment]
@rt4914 PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.
